LondonBritishLibrary/orient/BLorient657.xml

```diff
@@ -0,0 +1,213 @@
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.5">
+<locus target="135r"/>
+<title xml:lang="en" type="complete" ref="LIT2375Synaxa#Hamle">Ḥamle</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.6">
+<locus target="182v"/>
+<title xml:lang="en" type="complete" ref="LIT2375Synaxa#Nahase">Naḥāse</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.7">
+<locus target="214r"/>
+<title xml:lang="en" type="complete" ref="LIT2375Synaxa#Pagwemen">Ṗagʷǝmen</title>
+<textLang mainLang="gez"/>
+</msItem>
+</msContents>
+<physDesc>
+  <objectDesc form="Codex">
+      <supportDesc>
+          <support>
+              <material key="parchment"/>
+          </support>
+          <extent>
+              <measure unit="leaf">221</measure>
+              <measure unit="blank">2</measure>
+              <dimensions type="outer" unit="in">
+                  <height>19.875</height>
+                  <width>18</width>
+              </dimensions>
+              </extent>
+              <foliation>Two unnumbered blank leaf at the beginning.</foliation>
+        </supportDesc>
+
+      <layoutDesc>
+          <layout columns="3" writtenLines="30 33"/>
+      </layoutDesc>
+  </objectDesc>
+  <handDesc>
+    <handNote xml:id="h1" script="Ethiopic">
+    <desc>Written in a fine hand, of the <date notBefore="1600" notAfter="1667">17th Cent.</date>, before 1667.</desc>
+    </handNote>
+    </handDesc>
+
+<decoDesc>
+<decoNote type="drawing" xml:id="d1">
+<locus target="#1"/>
+<desc>A picture in outline.</desc>
+</decoNote>
+</decoDesc>
+
+    <additions>
+    <list>
+    <item xml:id="a1">
+    <locus from="2v"/>
+    <desc type="GuestText"><title type="complete">A short chronological section from Adam to Christ, in Amharic.</title></desc>
+    <q xml:lang="am"><seg part="F">ዘንተ፡ ዘአጽሐፈ፡ <persName role="patron" ref="PRS13283SahlaMikael">ሣህለ፡ ሚካኤል፡</persName> <placeName ref="LOC5662Semada">ዘስማዳ</placeName>፡ ወዘጸሐፈ፡
+    <persName role="scribe" ref="PRS13284GabraIyasus">ገብረ፡ ኢየሲስ፡</persName> ዘ<placeName ref="LOC6563Sada">ፀዳ፡</placeName> ዘተቀብረ፡ ቦቱ፡ ጻድቅ፡ ዮሐንስ፡ ንጉሠ፡ ነገሥት፡
+    አንቢበክሙ፡ እንዘ፡ ትፌጽሙ፡ ፩አቡነ፡ ዘበሰማያት፡ በእንተ፡ እግዚአብሔር፡ አብ።</seg></q>
+    </item>
+
+  <item xml:id="a2">
+      <locus target="#4r"/>
+      <desc type="OwnershipNote">The note states that the manuscript belongs to the church of <placeName ref="INS0101MadhaneAlam">Madḫāne ʿAlam</placeName>
+      </desc>
+      <q xml:lang="gez">ስንክሳር፡ ዘቅዱስ፡ መድኃኔ፡ ዓለም።</q>
+  </item>
+
+  <item xml:id="a3">
+  <locus target="#2r #220v #221r"/>
+  <desc>Hymns to John the Baptist.</desc>
+  </item>
+
+<item xml:id="a4">
+<locus target="#218"/>
+<desc>F. 218 is a later insertion containing the commemoration of king <persName ref="PRS10375Yohanne">John I</persName>, 15th of Ḥamle. It bears the name of <persName ref="PRS13286WaldaMaryam">Walda Māryām</persName> and
+his father <persName ref="PRS13285HabtaMaryam">Habta Māryām</persName>.</desc>
+</item>
+
```

Comment:
This means that the manuscript is a composite one. You wpuld need to rework the record and add ms_Parts, and occasionally sort out information pertaining to the whole mansucript or to its parts